ui/components/app/confirm-page-container/enableEIP1559V2-notice/enableEIP1559V2-notice.js

```diff
@@ -34,7 +34,9 @@ export default function EnableEIP1559V2Notice({ isFirstAlert }) {
 
   return (
     <Box
-      margin={[0, 4, 4, 4]}
+      marginRight={4}
+      marginBottom={4}
+      marginLeft={4}
       marginTop={isFirstAlert ? 4 : 0}
       className="enableEIP1559V2-notice"
     >
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props